apps/dg/components/case_table/case_table_controller.js

```diff
@@ -1018,14 +1021,80 @@ DG.CaseTableController = DG.ComponentController.extend(
       editAttributeFormula: function( iAttrID ) {
         var tDataContext = this.get('dataContext'),
             tRef = tDataContext && tDataContext.getAttrRefByID( iAttrID),
-            tAttrName = tRef && tRef.attribute.get('name'),
-            tAttrFormula = tRef && tRef.attribute.get('formula');
+            tAttrName = tRef && tRef.attribute.get('name');
+
         DG.assert( tRef && tAttrName, "editAttributeFormula() is missing the attribute reference or attribute name" );
         // for now we use the newAttribute() method which will replace one attribute formula with another
         // if the new attribute has the same name as the old.
+
+        var tAttrFormula = tRef.attribute.get('formula');
```

Comment:
I think I made this minor change for implementing the freeze / unfreeze. It should not affect the overall behaviors.

Reply:
Actually, this line can be put back to the original location. It was needed for automatically unfreezing the formula upon opening the edit-formula window, but obviously I removed the freezing stuff.

Reply:
I agree with putting it back to its original location. Cleaner.